lib/trino-plugin-toolkit/src/test/java/io/trino/plugin/base/security/TestFileBasedAccessControl.java

```diff
@@ -89,7 +93,7 @@ public void testEmptyFile()
         assertEquals(accessControl.filterTables(UNKNOWN, tables), tables);
 
         // permissions management APIs are hard coded to deny
-        TrinoPrincipal someUser = new TrinoPrincipal(PrincipalType.USER, "some_user");
```

Comment:
separate commit, for static imports.

Reply:
Oh, I did make this by accident, reverted